### ISSUE TITLE
Default docker-compose > validate > throwing: fping must have CAP_NET_RAW capability

### ISSUE THREAD

**Issue by Mist-Hunter** [librenms/docker#140]
### Behaviour
The docker-compose example is producing the following error (unless I'm the error)
Fail: /usr/sbin/fping could not be executed. /usr/sbin/fping must have CAP_NET_RAW capability (getcap) or suid. Selinux exclusions may be required. (/usr/sbin/fping: can't create socket (must run as root?))
Fail: /usr/sbin/fping6 could not be executed. /usr/sbin/fping6 must have CAP_NET_RAW capability (getcap) or suid. Selinux exclusions may be required. (/usr/sbin/fping: can't create socket (must run as root?))
Fail: /usr/sbin/fping6 should have CAP_NET_RAW!
Fix:
setcap cap_net_raw+ep /usr/sbin/fping6 

#### Steps to reproduce this issue

1.
git clone https://github.com/librenms/docker.git
2.
cd ./docker/examples/compose/
3.
docker-compose up -d
4. 
http://yoururl/validate

#### Expected behaviour
I'd expect only minor errors with the example docker-compose.yml, though maybe I've missed something.

Also note: the listed Fix
setcap cap_net_raw+ep /usr/sbin/fping6 

Doesn't seem to change anything, and fping is functioning properly.

> Tell me what should happen

#### Actual behaviour

> Tell me what happens instead

### Configuration

* Docker version (type `docker --version`) :
Docker version 19.03.13, build 4484c46d9d

* Docker compose version if applicable (type `docker-compose --version`) : 
docker-compose version 1.21.0, build unknown

* Platform (Debian 9, Ubuntu 18.04, ...) : 
Debian 10.6

* System info (type `uname -a`) : 
Linux <removed> 4.19.0-11-amd64 #1 SMP Debian 4.19.146-1 (2020-09-17) x86_64 GNU/Linux
* Include all necessary configuration files : `docker-compose.yml`, `.env`, ...

### Docker info

```
> Output of command `docker info`
```
Client:
 Debug Mode: false

Server:
 Containers: 10
  Running: 8
  Paused: 0
  Stopped: 2
 Images: 11
 Server Version: 19.03.13
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Native Overlay Diff: true
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 8fba4e9a7d01810a393d5d25a3621dc101981175
 runc version: dc9208a3303feef5b3839f4323d9beb36df0a9dd
 init version: fec3683
 Security Options:
  apparmor
  seccomp
   Profile: default
 Kernel Version: 4.19.0-11-amd64
 Operating System: Debian GNU/Linux 10 (buster)
 OSType: linux
 Architecture: x86_64
 CPUs: 4
 Total Memory: 2.781GiB
 Name: mistDockerGreen
 ID: GSBT:QKGZ:X7ZT:FQHG:U3D3:5OFK:3MOH:GODJ:N6ZU:SZTG:WKIT:NX6N
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: true

WARNING: No swap limit support

### Logs

```
> Container logs (set LOG_LEVEL to debug if applicable)
```

**Comment by crazy-max** [comment]
@Mist-Hunter I've pushed some changes, can you try with `librenms/librenms:edge` image? You may also need to add capability to your compose file: https://github.com/librenms/docker/blob/5bf8af707496f888bf258523308dd7d7ce0f51f9/examples/compose/docker-compose.yml#L48-L50

**Comment by minusdelta** [comment]
`NET_RAW` capability option is allowed by default: [docs](https://docs.docker.com/engine/reference/run/#runtime-privilege-and-linux-capabilities)

**Comment by crazy-max** [comment]
@minusdelta Yes that's true but NET_RAW capability will be removed in the near future (moby/moby#41030) so doesn't hurt to specify.

**Comment by Mist-Hunter** [comment]
Just want to add here that my problem was likely caused by a local setup. Adding here for others that may run into this.

If you have, "no-new-privileges": true in your /etc/docker/daemon.json, you will run into this issue. Removing this line from my Docker engine config resolved this issue. **Sorry for any work that was done, when this was on my end!!**

I added this line after doing a [Docker security audit](https://github.com/docker/docker-bench-security) and putting it into my base config. This is the third item I've had to chase down that breaks specific containers. Lesson: don't over do-it on security ;)

Found a comment in another LibreNMS container repo that put me on the right track. https://github.com/jarischaefer/docker-librenms/issues/90
